src/docfx/build/markdown/BuildMarkdown.cs

```diff
@@ -2,17 +2,19 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace Microsoft.Docs.Build
 {
     internal static class BuildMarkdown
     {
-        public static Task Build(Context context, Document file, TableOfContentsMap tocMap, Action<Document> buildChild)
+        public static Task<Dictionary<Document, IEnumerable<DependencyItem>>> Build(Context context, Document file, TableOfContentsMap tocMap, Action<Document> buildChild)
```

Comment:
>Dictionary<Document, IEnumerable<DependencyItem>> [](start = 27, length = 49)

suggest to create a container struct `DependencyMap`, this type declaration does not look obvious

Reply:
it's a one to many mapping, i think it's clear enough

---
In reply to: [193959424](https://github.com/dotnet/docfx/pull/2859#discussion_r193959424) [](ancestors = 193959424)